### PR TITLE
Feature :Added Image Component

### PR DESCRIPTION
**Pull request by kiranpratyush**
updated index.html according to the documentation site(though some changes are pending for header ignore it for now updated the style.css
 Below is the live working link please review it and give help full feedback

Components Defined - 
- Image

Live Link - 
Documentation site : [Testing](https://testing2311.netlify.app/image/index.html)

### CODE REVIEW

**Review by jayeshjain98 — APPROVED** [review]
looks clean

**Comment by kiranpratyush** [comment]
> looks clean

Thank you so much for reviewing

**Comment by kiranpratyush** [comment]
> looks clean

Hey Thank you so much for taking the time to review it
I have updated the code and index.html according to my documentation website kindly please go through it and give valuable feedback

File: image/index.html [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+        >
+          <iframe
+            src="https://carbon.now.sh/embed?bg=rgba%28249%2C237%2C212%2C1%29&t=twilight&wt=none&l=htmlmixed&width=680&ds=false&dsyoff=0px&dsblur=4px&wc=true&wa=false&pv=0px&ph=0px&ln=false&fl=1&fm=IBM+Plex+Mono&fs=18px&lh=160%25&si=false&es=2x&wm=false&code=%253Cimg%2520src%253D%2522.%252Fimg1.jpg%2522%2520alt%253D%2522%2522%2520class%253D%2522circle%2522%2520%252F%253E"
+            style="
+              width: 680px;
+              height: 103px;
+              border: 0;
+              transform: scale(1);
+              overflow: hidden;
+            "
```
Comment (tanveertkd) [inline image/index.html:84]: You can move the iFrame styling to one of the common stylesheets.

**Review by tanveertkd — APPROVED** [review]
Just the one suggestion. Otherwise it's good to go.

File: image/index.html (hunk 1 above)
Comment (kiranpratyush) [reply image/index.html:84]: Thanks, Tanveer for your valuable feedback .

**Pull request merged**